libraries/AP_HAL_Linux/SPIUARTDriver.cpp

```diff
@@ -193,7 +186,7 @@ int SPIUARTDriver::_write_fd(const uint8_t *buf, uint16_t size)
     
 }
 
-static const uint8_t ff_stub[300] = {0xff};
+static uint8_t ff_stub[300] = {0xff};
```

Comment:
maybe you could use a struct here with the first element as `const` and other `299` elements to be filled